tests/clone_test.go

```diff
@@ -653,6 +653,48 @@ var _ = Describe("[Serial][sig-compute]VirtualMachineClone Tests", Serial, decor
 					Expect(libinstancetype.EnsureControllerRevisionObjectsEqual(sourceVM.Spec.Preference.RevisionName, targetVM.Spec.Preference.RevisionName, virtClient)).To(BeTrue(), "source and target preference controller revisions are expected to be equal")
 				})
 			})
+
+			It("double cloning: clone target as a clone source", func() {
+				addAnnotationAndLabelFilters := func(vmClone *clonev1alpha1.VirtualMachineClone) {
+					filters := []string{"somekey/*"}
+					vmClone.Spec.LabelFilters = filters
+					vmClone.Spec.AnnotationFilters = filters
+				}
+				generateCloneFromVMWithParams := func(sourceVM *virtv1.VirtualMachine, targetVMName string) *clonev1alpha1.VirtualMachineClone {
+					vmclone := generateCloneFromVMWithParams(sourceVM, targetVMName)
+					addAnnotationAndLabelFilters(vmclone)
+					return vmclone
+				}
+
+				snapshotStorageClass, err := libstorage.GetSnapshotStorageClass(virtClient)
+				Expect(err).ToNot(HaveOccurred())
+				if snapshotStorageClass == "" {
+					Skip("Skipping test, no VolumeSnapshot support")
```

Comment:
Any chance we could introduce a decorator for this instead of skipping?

Reply:
Thank you @jean-edouard!
This comment made me find out that half of the tests are being silently skipped :facepalm: 

Thanks to your comment now all of the skips are removed and new decorators are introduced. Thanks!

Reply:
The storage lanes now run the newly introduced test :)